pom.xml

```diff
@@ -34,6 +34,268 @@
+        <!-- Community plugins -->
+        <gravitee-connector-kafka.version>1.1.1</gravitee-connector-kafka.version>
+        <gravitee-policy-aws-lambda.version>1.2.0</gravitee-policy-aws-lambda.version>
+        <gravitee-policy-circuit-breaker.version>1.1.5</gravitee-policy-circuit-breaker.version>
+        <gravitee-policy-geoip-filtering.version>2.0.0</gravitee-policy-geoip-filtering.version>
+        <gravitee-policy-javascript.version>1.3.3</gravitee-policy-javascript.version>
+        <gravitee-resource-auth-provider-http.version>1.4.0</gravitee-resource-auth-provider-http.version>
+        <gravitee-resource-auth-provider-inline.version>1.4.0</gravitee-resource-auth-provider-inline.version>
+        <gravitee-resource-auth-provider-ldap.version>1.4.0</gravitee-resource-auth-provider-ldap.version>
+        <gravitee-resource-cache-redis.version>3.0.0-alpha.1</gravitee-resource-cache-redis.version>
+        <gravitee-resource-oauth2-provider-keycloak.version>2.1.0</gravitee-resource-oauth2-provider-keycloak.version>
+        <gravitee-service-geoip.version>3.0.0</gravitee-service-geoip.version>
+        <gravitee-secretprovider-kubernetes.version>2.0.0-alpha.2</gravitee-secretprovider-kubernetes.version>
+
+        <!-- Enterprise plugins -->
+        <gravitee-entrypoint-http-get.version>2.0.0-alpha.2</gravitee-entrypoint-http-get.version>
+        <gravitee-entrypoint-http-post.version>2.0.0-alpha.2</gravitee-entrypoint-http-post.version>
+        <gravitee-entrypoint-sse.version>5.0.0-alpha.2</gravitee-entrypoint-sse.version>
+        <gravitee-entrypoint-webhook.version>4.0.0-alpha.2</gravitee-entrypoint-webhook.version>
+        <gravitee-entrypoint-websocket.version>2.0.0-alpha.2</gravitee-entrypoint-websocket.version>
+        <gravitee-endpoint-kafka.version>3.0.0-alpha.3</gravitee-endpoint-kafka.version>
+        <gravitee-endpoint-mqtt5.version>3.0.0-alpha.3</gravitee-endpoint-mqtt5.version>
+        <gravitee-endpoint-rabbitmq.version>2.0.0-alpha.5</gravitee-endpoint-rabbitmq.version>
+        <gravitee-endpoint-solace.version>2.0.0-alpha.3</gravitee-endpoint-solace.version>
+        <gravitee-endpoint-azure-service-bus.version>1.0.0-alpha.3</gravitee-endpoint-azure-service-bus.version>
+        <gravitee-policy-graphql-rate-limit.version>1.0.1</gravitee-policy-graphql-rate-limit.version>
+        <gravitee-resource-schema-registry-confluent.version>3.1.0</gravitee-resource-schema-registry-confluent.version>
+        <gravitee-reactor-message.version>5.0.0-alpha.11</gravitee-reactor-message.version>
+        <gravitee-reactor-native-kafka.version>1.0.0-alpha.34</gravitee-reactor-native-kafka.version>
+        <gravitee-apim-repository-bridge.version>5.1.2</gravitee-apim-repository-bridge.version>
+        <gravitee-secretprovider-hc-vault.version>2.0.0-alpha.4</gravitee-secretprovider-hc-vault.version>
+        <gravitee-secretprovider-aws.version>2.0.0-alpha.2</gravitee-secretprovider-aws.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secretprovider-aws.version>*************</gravitee-secretprovider-aws.version>  
[This comment was created by Aqua Pipeline]